atlas/additional-guides/troubleshooting-builds.mdx

```diff
@@ -67,6 +67,22 @@ After the platform pulls your code and runs `npm install`, it will execute the `
   },
 ```
 
+### Make sure the branch and repository you're pulling from exists
+
+If you've deleted your branch or repository, you'll see the following message on the environment details screen:
+<img width="599" alt="image" src="https://user-images.githubusercontent.com/15729608/177279683-d43d93a3-30f7-4bdc-b883-5f79ab158f92.png">
+
+You will also see an alert next to the branch name:
+![image](https://user-images.githubusercontent.com/15729608/177280170-92c651db-d4da-4a58-bec4-d6b81aa8e416.png)
+
+In csae of deleted branch (or if you simply want to switch a branch your environment is pulling code from to, for example, a feature branch), you can do it by clicking on the "edit" button next to the branch name, selecting a branch and clicking "Save":
```

Comment:
Small typos, "csae" -> "case," "edit" -> "Edit." And maybe a small wording change to this step's intro sentence: If you accidentally delete a branch, or if you simply want to switch the branch your environment is pulling code from a production branch to a feature branch, for example), you can do it by clicking on the "Edit" button next to the branch name, selecting a new branch and clicking "Save":